runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperConfig.java

```diff
@@ -71,21 +71,54 @@ default boolean quickhopOutOfDanger()
 	}
 
 	@ConfigItem(
-		keyName = "quickHopRegionFilter",
-		name = "Quick-hop region",
-		description = "Limit quick-hopping to worlds of a specific region",
+		keyName = "quickHopRegionFilterAU",
+		name = "Quick-hop to Australia worlds (AUS)",
+		description = "Quick-hop to worlds located in Australia",
 		position = 3
 	)
-	default RegionFilterMode quickHopRegionFilter()
```

Comment:
Since `RegionFilterMode` is used only for this configuration, the class can be removed with this PR as well.